app/presenters/service_presenter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class ServicePresenter < SimpleDelegator
+  def as_json(*)
+    {service: stringify_nil_values(filtered_values(__getobj__))}.merge(error_messages(__getobj__))
+  end
+
+  private
+
+  def filtered_values(hash)
+    hash.as_json(root: false, only: %i[name system_name description])
+  end
+
+  def error_messages(hash)
+    {errors: hash.errors.any? ? hash.errors.messages : {}}.as_json
+  end
+
+  def stringify_nil_values(hash)
+    hash.map {|key, value| [key, value.presence ? value : '']}.to_h
```

Comment:
all those value are string right? So you can just use 
```suggestion
    hash.transform_values(&:to_s)
```

Reply:
@hery those might or might not be `nil` values. The objective is to avoid feeding JS null values

Reply:
This will enforce them to be all `String` values `nil.to_s => ''`

Reply:
Do not ping `@hery` because it is not me @didierofrivia